### PR TITLE
Graph drawing saving feature

### PR DESCRIPTION
**Pull request by EugenKoulik**

### CODE REVIEW

**Review by dnbss — APPROVED** [review]

File: Forms/DrawForm/ToolPanel/SaveButton.cs [hunk 1]
```diff
@@ -51,13 +56,83 @@ public void ButtonClick(object sender, EventArgs e)
...
+            }
           
         }
+
+        private void HidingСontrols()
+        {
+
+            drawForm.FormBorderStyle = FormBorderStyle.None;
+
+            foreach (Control c in drawForm.Controls)
```
Comment (Maksim-Trolina) [inline Forms/DrawForm/ToolPanel/SaveButton.cs]: You can use var instead of Control

Comment (Maksim-Trolina) [inline Forms/DrawForm/ToolPanel/SaveButton.cs]: Instead of c, you can use control as a name

File: Forms/DrawForm/ToolPanel/SaveButton.cs [hunk 2]
```diff
@@ -51,13 +56,83 @@ public void ButtonClick(object sender, EventArgs e)
...
+
+        private void ShowingСontrols()
+        {
+
+            drawForm.FormBorderStyle = FormBorderStyle.FixedDialog;
+
+            foreach (Control c in drawForm.Controls)
+            {
+
+                if ((bool)c.Tag)
```
Comment (Maksim-Trolina) [inline Forms/DrawForm/ToolPanel/SaveButton.cs:99]: Isn't c.Tag a bool type?

File: Forms/DrawForm/ToolPanel/SaveButton.cs [hunk 3]
```diff
@@ -51,13 +56,83 @@ public void ButtonClick(object sender, EventArgs e)
...
+        {
+
+            Size szCurrent = control.Size;
+            control.AutoSize = true;
+
+            Rectangle rectangle = new Rectangle();
+
+            rectangle.Width = control.Width;
+
+            rectangle.Height = control.Height;
```
Comment (Maksim-Trolina) [inline Forms/DrawForm/ToolPanel/SaveButton.cs]: Can be done in one line

**Review by Maksim-Trolina — APPROVED** [review]

**Pull request merged**